miden/src/examples/mod.rs

```diff
@@ -1,6 +1,6 @@
 use assembly::diagnostics::{IntoDiagnostic, Report};
 use clap::Parser;
-use miden_vm::{ExecutionProof, Host, Program, ProgramInfo, ProvingOptions, StackInputs};
+use miden_vm::{ExecutionProof, Host, MastForestInfo, Program, ProvingOptions, StackInputs};
```

Comment:
I think `MastForestInfo` may be an old name - I can't find it any more.

Reply:
Fixed, along with all the errors under `miden-vm`.

Turns out, most of the code there is under the `executable` feature, and so didn't get checked when I ran `cargo check` at the root. And our CI also doesn't check it. We should fix that

Reply:
Created #1354